### DISCUSSION TITLE
Small Wiki Link Fix

### DISCUSSION THREAD

**Discussion by albino1** [tophf/mpiv#70] [MPIV general]
On the [Home Wiki Page](https://github.com/tophf/mpiv/wiki) on the part where it says "report a problem with the existing **ones**" the "ones" part links to a 404:

https://github.com/tophf/mpiv/Rules

The correct link should be:

https://github.com/tophf/mpiv/wiki/Rules

Just starting using this, looks great.  Hope this helps :)

**Comment by darkred** [comment]
Fixed, thank you.

**Comment by nomadism19** [comment]
Cannot find code to overlay image caption over top-right of image (not top-right of window). Excuse me if this is wrong place to ask.

**Reply by tophf** [reply]
Not possible.
